src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -180,4 +183,43 @@ public ReachAngle(double rotationInDegrees) {
 	public String getName() {
 		return name;
 	}
+	static class ConcurrentActions implements Action.Activity{
+		List<Action.Activity> activities = new ArrayList<Action.Activity>();
```

Comment:
Leave activities unassigned here. It gets assigned in the constructor.

Reply:
You missed this one, there's no need to allocate an ArrayList here, it gets replaced in the constructor.